contracts/Archetype.sol

```diff
@@ -433,7 +438,7 @@ contract Archetype is ERC721A__Initializable, ERC721AUpgradeable, ERC721A__Ownab
   }
```

Comment:
can we rename the `reveal()` function to something more accurate like `toggleRevealed()`? Currently misleading

Reply:
not sure if we even need it anymore, it is part of an old design pattern we don't use anymore